packages/cdk-routing-split/serverless.yml

```diff
@@ -38,3 +62,33 @@ custom:
         number: 3
     modularize:
         glob: 'src/**/serverless.module.yml'
+
+resources:
+    Resources:
+        dynamoDbTable:
+            Type: AWS::DynamoDB::Table
+            Properties:
+                SSESpecification:
+                    SSEEnabled: true
```

Comment:
@nthornton2010 were you suggesting in the jira story that this was not needed?

Reply:
@BennuAshes SSEEnabled is required. I was saying that SSEType is not needed as we'd use the default value.

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-dynamodb-table-ssespecification.html